04_snake_game/main.go

```diff
@@ -123,27 +193,34 @@ func main() {
 			eventQueue <- termbox.PollEvent()
 		}
 	}()
+
+	ticker := time.NewTicker(70 * time.Millisecond)
+	defer ticker.Stop()
+	drawborder()
```

Comment:
The border is shown only during the game start. It dissapears afterwards. Can you please fix it?